### PR TITLE
Create dockerfile for build if needed.

### PR DESCRIPTION
**Pull request by wenottingham**
See https://github.com/ansible/awx/pull/7017

### CODE REVIEW

**Comment by wenottingham** [comment]
recheck

**Comment by TristanCacqueray** [comment]
Linters job should be fixed by https://github.com/ansible/zuul-config/pull/85

File: roles/run_awx_tests/tasks/main.yml [hunk 1]
```diff
@@ -16,6 +16,13 @@
     docker pull gcr.io/ansible-tower-engineering/awx_devel:{{ target_branch }}
   ignore_errors: yes
 
+- name: Create docker file if necessary
+  template:
+    src: installer/roles/image_build/templates/Dockerfile.j2
```
Comment (shanemcd) [inline roles/run_awx_tests/tasks/main.yml:21]: I think both of these need to be prepended with: `{{ source_dir }}`. See where we use that below.

**Comment by shanemcd** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

**Pull request merged**